src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1339,7 +1338,7 @@ bool FancyZones::ShouldProcessSnapHotkey(DWORD vkCode) noexcept
         }
 
         auto zoneWindow = m_workAreaHandler.GetWorkArea(m_currentDesktopId, monitor);
-        if (zoneWindow && zoneWindow->ActiveZoneSet() != nullptr)
+        if (zoneWindow && zoneWindow->ActiveZoneSet() && zoneWindow->ActiveZoneSet()->LayoutType() != FancyZonesDataTypes::ZoneSetLayoutType::Blank)

```

Comment:
Maybe it would be better to check `zoneWindow->ActiveZoneSet()` before `FancyZonesUtils::IsCandidateForZoning(window, m_settings->GetSettings()->excludedAppsArray)`

Reply:
The behavior would be equivalent and the impact on performance would be negligible.
Also, see the commit I'm adding right now for why the current order makes more sense.

Reply:
OK